src/Command/ServeCommand.php

```diff
@@ -54,21 +54,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $cacheDir = $this->container->getParameter('mate.cache_dir');
         \assert(\is_string($cacheDir));
 
-        $envFile = $this->container->getParameter('mate.env_file');
-
-        // 0. Discover extensions with their filters and service files
+        // Discover extensions with their filters and service files
         $extensions = $this->getExtensionsToLoad();
 
-        // 1. Load environment variables from .env files
-        if (null !== $envFile && \is_string($envFile)) {
-            $extra = [];
-            $localFile = $rootDir.\DIRECTORY_SEPARATOR.$envFile.\DIRECTORY_SEPARATOR.'.local';
-            if (!file_exists($localFile)) {
-                $extra[] = $localFile;
-            }
-            (new Dotenv())->load($rootDir.\DIRECTORY_SEPARATOR.$envFile, ...$extra);
-        }
-
         // 2. Create filtered discovery loader
```

Comment:
```suggestion
        // Create filtered discovery loader
```

maybe also reming all the numbering in the comments

Reply:
Yeah. I am working my way on it.